core-libs/setup/ssr/error-handling/server-error-response-transformers/cms-page-not-found-server-error-response-transformer.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { ServerErrorResponseTransformer } from './server-error-response-transformers';
+import { CmsPageNotFoundServerError, CxServerError } from '../server-errors';
+import { HttpErrorResponse } from '@angular/common/http';
+import { Priority } from '@spartacus/core';
+
+/**
+ * A transformer responsible for transforming an HTTP error response into a {@link CmsPageNotFoundServerError}.
+ * The transformer is applicable when the error the URL contains 'cms/pages'.
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class CmsPageNotFoundServerErrorResponseTransformer
+  implements ServerErrorResponseTransformer
+{
+  hasMatch(error: unknown): boolean {
+    return (
+      this.isHttpErrorResponse(error) && !!error.url?.includes('cms/pages')
+    );
+  }
+
+  getPriority(): number {
+    return Priority.NORMAL;
```

Comment:
Let's use `LOW` by default in our library, so customers' ones have higher priority by default (`NORMAL`)